src/components/MainForm/Mainform.tsx

```diff
@@ -187,8 +177,11 @@ export default function Mainform() {
               })}
               style={{ marginBottom: "20px" }}
             />
-            {errors.email && <span id="error-msg">Invalid email address</span>}
-            {/* </Box> */}
+            {errors.email && (
```

Comment:
you don't need to do this!!
See here to show error messages
https://mui.com/components/text-fields/#validation

Reply:
@ZainShahbaz implement this

Reply:
done